ee/webhooks/README.md

```diff
@@ -12,26 +12,16 @@ The service has 3 starting modes, split into 3 separate commands:
 - `server`: REST web service API managing webhooks configs for users.
 - `worker`: background service consuming kafka events on selected topics to send webhooks based on user configs and periodically finding failed webhooks requests to retry and sending new attempts.
 
-## Run the stack locally
-```
-docker compose up
-```
-
-## Run linters and tests locally
-
-```
-task install:lint
-task
-```
+## Run linters and tests 
 
-Run the tests for a specific package:
+Run the linters:
 ```
-task tests:local PKG=./pkg/model
+earthly +lint
```

Comment:
Should we add -P for priviledge ? (for users that do not have them in their configs)

Reply:
For the +tests, we do not need it for lint sorry

Reply:
I added -P for the `earthly -P +tests` command.